cpg_workflows/stages/large_cohort.py

```diff
@@ -31,6 +31,8 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
 
         j = get_batch().new_job('Combiner', (self.get_job_attrs() or {}) | {'tool': 'hail query'})
 
+        init_batch_args = {'worker_memory': 'highmem'} if get_config()['workflow'].get('highmem_works') else None
```

Comment:
do you want to add `highmem_works` to the default large_cohort config?

Reply:
^ second this

Reply:
Added!